src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -278,7 +278,7 @@ internal BoundExpression BindToNaturalType(BoundExpression expression, Diagnosti
                         sourceTuple.Syntax,
                         sourceTuple,
                         wasTargetTyped: false,
-                        sourceTuple.Arguments.SelectAsArray(e => BindToNaturalType(e, diagnostics, reportDefaultMissingType)),
+                        sourceTuple.Arguments.SelectAsArray((e, arg) => arg.Item1.BindToNaturalType(e, arg.diagnostics, arg.reportDefaultMissingType), (this, diagnostics, reportDefaultMissingType)),
```

Comment:
>Item1 [](start = 76, length = 5)

Please use more descriptive name as for other items in the same tuple.

Reply:
Done